src/components/reviews/FilterRatings.jsx

```diff
@@ -1,9 +1,75 @@
-import React from 'react';
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import { getProductRating, getRatings } from './api.js';
+import SingleRating from './SingleRating.jsx';
+
+function FilterRatings(props) {
+  const [rating, setRating] = useState();
+
+  const { filter, setFilter, filters } = props;
+  const { selectedFilters, setSelectedFilters } = props;
+
+  const product = useSelector((state) => state.product.productId);
+
+  // get producr rating on render
+  useEffect(() => {
+    getProductRating(product)
+      .then((result) => {
+        setRating(result);
+      });
+  }, []);
```

Comment:
For `useEffect`s that send API calls based on a product ID, we also want to make sure that we send new API calls if that ID changes, which can be done by adding `product` to the `useEffect` array.

Reply:
great catch, thank you!